File: src/test/java/parse/NotepadTesting.java
package parse;

import util.Notepad.NotepadProcessor;

public class NotepadTesting {
    public static void main(String[] args) {
        NotepadProcessor np = new NotepadProcessor();
        try {
            np.openNotepad();
            np.writeText("Hello World");
            np.addNewLine();
            np.writeText("This is a test");
            np.saveFileAs("test.txt");
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: src/test/java/parse/NotepadTesting.kt
package parse

import util.notepad.NotepadProcessor

fun main() {
  NotepadProcessor().apply {
    try {
      openNotepad()
      writeText("Hello World\nThis is a test")
      saveFileAs("test.txt")
    } catch (e: Exception) {
      e.printStackTrace()
    }
  }
}